prov/efa/test/efa_unit_test_runt.c

```diff
@@ -127,6 +127,53 @@ void test_efa_rdm_peer_get_runt_size_cuda_memory_normal(struct efa_resource **st
 	test_efa_rdm_peer_get_runt_size_impl(resource, FI_HMEM_CUDA, peer_num_runt_bytes_in_flight, total_runt_size, msg_length, expected_runt_size);
 }
 
+/* When using LL128 protocol, the segmented size of runting read must be 128 multiple. */
```

Comment:
do you need some more test cases when the available runting bytes is bigger than the total len?

Reply:
Also need a test case when the available runting bytes is smaller than 128.